wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/Container/ViewModel/ConversationListViewControllerViewModel+StartUIDelegate.swift

```diff
@@ -45,14 +45,22 @@ extension ConversationListViewController.ViewModel: StartUIDelegate {
         _ user: UserType,
         callback onConversationCreated: @escaping ConversationCreatedBlock
     ) {
-        guard let userSession = ZMUserSession.shared() else { return }
+        guard let userSession = ZMUserSession.shared() else {
+            return
+        }
+
+        if let conversation = user.oneToOneConversation {
+            onConversationCreated(.success(conversation))
+        } else {
+            viewController?.setState(.conversationList, animated: true) {
+                userSession.createTeamOneOnOne(with: user) {
+                    switch $0 {
```

Comment:
For reading it's easier to keep the same level of abstraction like line 53, so I would suggested to extract a func and call it in line 55.

Reply:
done